packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx

```diff
@@ -22,13 +23,18 @@ export const DashboardExtension = ({
   const location = useLocation();
   const navLink = useMemo(() => decodeURIComponent(last(location.pathname.split('/'))), [location.pathname]);
 
+  const renderIcon = title === 'Vitals & Biometrics' ? <Activity style={{ marginRight: '8px' }} /> : null;
```

Comment:
Can we use [ActivityIcon](https://github.com/openmrs/openmrs-esm-core/blob/69859408fa1b7bf773a89002b852cab79a1ae309/packages/framework/esm-styleguide/src/icons/icons.tsx#L14) here instead?